UnityMcpBridge/UnityMcpServer~/src/tools/resource_tools.py

```
@@ -293,57 +318,54 @@ async def read_resource(
-                    "success": True,
-                    "data": {"text": text, "metadata": {"sha256": sha}},
-                }
-            return {
-                "success": True,
-                "data": {"metadata": {"sha256": sha, "lengthBytes": length}},
-            }
+                return {"success": True, "data": {"text": text, "metadata": {"sha256": full_sha, "lengthBytes": len(full_bytes)}}}
+            else:
+                # Default: metadata only
+                return {"success": True, "data": {"metadata": {"sha256": full_sha, "lengthBytes": len(full_bytes)}}}
         except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)